.changeset/thirty-moose-allow.md

```diff
@@ -0,0 +1,6 @@
+---
+"gitbook-docs": patch
+"nextjs-website": patch
```

Comment:
I think it's a minor, not a patch because you are adding a new non-breaking feature